Code/Editor/EditorViewportSettings.h

```diff
@@ -166,4 +168,7 @@ namespace SandboxEditor
 
     SANDBOX_API float CameraDefaultFarPlaneDistance();
     SANDBOX_API void SetCameraDefaultFarPlaneDistance(float distance);
+
+    SANDBOX_API float CameraDefaultFovRadians();
+    SANDBOX_API void SetCameraDefaultFovRadians(float fov);
```

Comment:
update: It would be good to have the degrees versions too that do the right conversions